src/Repository/OrderRepository.php

```diff
@@ -41,17 +41,17 @@ public function orderCanBeCancelled($idCart, $currentState)
         $sql = 'SELECT COUNT(DISTINCT(opn.`id_oyst_payment_notification`))'
             .' FROM `'._DB_PREFIX_.'oyst_payment_notification` opn'
             .' WHERE opn.`id_cart` = '.(int) $idCart
-            .' AND opn.`event_code` = "'. OystPaymentNotification::EVENT_AUTHORISATION.'"'
+            .' AND opn.`event_code` = "'. \OystPaymentNotification::EVENT_AUTHORISATION.'"'
             .' AND opn.`id_cart` NOT IN ('
                 .'SELECT opn_bis.`id_cart`'
                 .' FROM `'._DB_PREFIX_.'oyst_payment_notification` opn_bis'
-                .' WHERE opn_bis.`event_code` = "'.OystPaymentNotification::EVENT_CAPTURE.'"'
-                .' OR opn_bis.`event_code` = "'.OystPaymentNotification::EVENT_CANCELLATION.'"'
+                .' WHERE opn_bis.`event_code` = "'.\OystPaymentNotification::EVENT_CAPTURE.'"'
```

Comment:
Maybe is better to import the class directly ? :) And for duplicate one, use the rename class 
Ex use Configuration as PSConfiguration